src/v2/guide/single-file-components.md

```diff
@@ -1,48 +1,48 @@
+Como prometimos, podemos también usar preprocesadores como Pug, Babel (con módulos ES2015) y Stylus para obtener componentes más limpios y con más funcionalidades.
 
-<a href="https://gist.github.com/chrisvfritz/1c9f2daea9bc078dcb47e9a82e5f7587" target="_blank"><img src="/images/vue-component-with-preprocessors.png" alt="Single-file component example with preprocessors (click for code as text)" style="display: block; margin: 30px auto;"></a>
+<a href="https://gist.github.com/chrisvfritz/1c9f2daea9bc078dcb47e9a82e5f7587" target="_blank"><img src="/images/vue-component-with-preprocessors.png" alt="Ejemplo de Componente de un Solo Archivo con preprocesadores (click para ver código como texto)" style="display: block; margin: 30px auto;"></a>
 
-Estos lenguajes específicos son solo ejemplos. Usted podrías usar facilmente Bublé, TypeScript, SCSS, PostCSS - o cualquier otro preprocesador que le ayude a ser productivo. Si utiliza Webpack con `vue-loader`, esto también tiene soporte de primera clase para módulos CSS.
+Estos lenguajes específicos son solo ejemplos. Podrías usar, con la misma facilidad, Bublé, TypeScript, SCSS, PostCS, o cualquier otro preprocesador que te ayude a ser productivo. Si utilizas Webpack con `vue-loader`, también te ofrece soporte de primera clase para módulos CSS.
 
-### Qué sobre la separación de
+### ¿Qué sucede con la separación de intereses?
 
-Una cosa importante a notar es que **la separación de concerns no es igual a la separación de tipos de archivo.** En el desarrollo moderno de UI, nos encontramos que en lugar de dividir el codigo base dentro de 3 capaz enormes que se entrelazan entre si, tiene mucho más sentido dividirlas dentro de componentes más livianos y componerlos entre sí.  Dentro de un componente, está su plantilla, lógica y los estilos son inherentemente acoplados, y su implementación hace que el componente sea más cohesivo y mantenible.
+Es importante notar que **la separación de intereses no equivale a la separación de tipos de archivo.** En el desarrollo moderno de UI nos encontramos con que, en lugar de dividir el código base dentro de 3 capas enormes que se entrelazan entre sí, tiene mucho más sentido dividirlas dentro de componentes más livianos, y componerlos las capas sí. Dentro de un componente, su plantilla, lógica y estilos están inherentemente acoplados y ubicándolos en un mismo lugar logramos que el componente sea más coherente y mantenible.
 
-Incluso si no le gusta la idea de utilizar Single-File Components, usted puede aprovechar las características de hot-reloading y pre-compilación separando su JavaScript y CSS en archivos diferentes:
+Incluso si no te gusta la idea de utilizar Componentes de un Solo Archivo, puedes aprovechar las características de recarga en caliente y compilación previa, separando tu JavaScript y CSS en archivos diferentes:
```

Comment:
I think we are using `le` over `te`, right @sxcamacho 
Also, I'm not sure if we should translate `hot-reload`. What do you guys think @sxcamacho @FLarra

Reply:
You're right. We should only point the translation in the `"usted"` form.

I prefer `hot-reloading` but I'm not 100% we have stated a preference for this word.

Reply:
Changed to formal treatment throughout the document.